airflow/jobs.py

```diff
@@ -695,18 +714,19 @@ def _execute(self):
                     dags = [
                         dag for dag in dagbag.dags.values()
                         if not dag.parent_dag]
+
                 paused_dag_ids = dagbag.paused_dags()
-                for dag in dags:
-                    self.logger.debug("Scheduling {}".format(dag.dag_id))
-                    dag = dagbag.get_dag(dag.dag_id)
-                    if not dag or (dag.dag_id in paused_dag_ids):
-                        continue
-                    try:
-                        self.schedule_dag(dag)
-                        self.process_dag(dag, executor)
-                        self.manage_slas(dag)
-                    except Exception as e:
-                        self.logger.exception(e)
+                dags = [x for x in dags if x.dag_id not in paused_dag_ids]
+                self.logger.info("CPU: {} DAGs:{}".format(multiprocessing.cpu_count(), len(dags)))
```

Comment:
How expensive is cpu_count? I notice we're calling it multiple times in the scheduler loop.